### DISCUSSION TITLE
[`@types/node`] `Stats` is deprecated

### DISCUSSION THREAD

**Discussion by AlttiRi** [DefinitelyTyped/DefinitelyTyped#69713] [Issues with a @types package]
In `"@types/node": "20.12.13"` `Stats`  is marked as deprecated, while only `Stats`' `constructor` was deprecated in https://github.com/nodejs/node/pull/51879.

![Screenshot](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/16310547/299740f8-74c5-4c61-ba84-abb94314b522)

```ts
import fs from "node:fs/promises";
import {Stats, BigIntStats} from "node:fs";

let anyStats: Stats | BigIntStats;
const stats: Stats = await fs.stat(".");
anyStats = stats;
console.log(anyStats);
```

`"@types/node": "20.12.12"` is OK.

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
This was a mistake on my end, will revert this as anyway the changes were for v20.13.1 but we're mistakenly published as 20.12.13 because I forgot to bump the version.
